hackathon-001/index.html

```diff
@@ -0,0 +1,123 @@
+				Setup your development environment, join us on Discord, and spend the weekend tinkering with the smartwatch platform that fans refuse to let die. <br><br>
+				There will be stickers sent out to <i>anyone</i> who submits a new app to the store, and a prize for our overall favourite submission(s).
+			</p>
+		</div>
+
+		<!-- Our list -->
+		<div class="ml break">
+			<h2 class="nomargin"> Our Preflight Checklist</h2>
+			(Rebble's To-Do list:)
+
+			<p class="hackathon-about" style="margin-left: 30px">
+				<i class="far fa-check-square"></i> Choose a date <br>
+				<i class="far fa-check-square"></i> Announce the hackathon! <br>
+				<i class="far fa-check-square"></i> <a href="./vm">Provide a VM image for easy developing </a><br>
+				<i class="far fa-square"></i> Spread the word <br>
+				<i class="far fa-square"></i> Run a great event <br>
+				<i class="far fa-square"></i> Send out stickers and other goodies to participants <br>
+			</p>
+		</div>
+
+		<!-- Your list -->
+		<div class="ml break">
+			<h2 class="nomargin"> Your Preflight Checklist</h2>
+
+			<p class="hackathon-about" style="margin-left: 30px">
+				<i class="far fa-square"></i> <a href="https://forms.gle/oACS5hbPedeiEwiG8" target="_blank">Confirm attendance</a> <br>
+				<i class="far fa-square"></i> <a href="./vm" target="_blank">Setup your development environment </a><br>
+				<i class="far fa-square"></i>  <a href="https://dev-portal.rebble.io/" target="_blank">Create a developer account </a><br>
+				<i class="far fa-square"></i> Wait patiently for <span id="countdown-small"></span> <br>
+				<i class="far fa-square"></i> Join us <a href="/discord" target="_blank">on Discord</a> and have fun! <br>
+			</p>
+		</div>
+
+
+		<!-- FAQ -->
+		<div class="break ml">
+
+			
+			<h1> FAQ </h1>
+			<h2> What is this? </h2>
+			<p> 
+				A <a href="https://en.wikipedia.org/wiki/Hackathon">hackathon</a>! A weekend where Pebble enthusiasts can get together (virtually) and build something cool for the Pebble platform.
+			</p>
+
+			<h2> How do I sign up? </h2>
+			<p> 
+				All you have to do is <a href="https://forms.gle/oACS5hbPedeiEwiG8">fill out this form</a>. You should also <a href="/discord">join us on Discord</a> too! We'll have dedicated channels during the hackathon.
+			</p>
+
+			<h2> How do I setup the SDK? </h2>
+			<p> 
+				The Pebble SDK is old, so the easiest way is use the <a href="./vm">official hackathon VM image</a>. For other installation instructions, <a href="https://help.rebble.io/sdk">see here.</a>
+			</p>
+
+			<h2> How long is the event? </h2>
+			<p> 
+				It starts Friday evening and runs until Sunday evening, across the 18th - 20th of November. You don't need to be sat at your computer the entire time! Just work on your project when you can, and drop by on Discord to chat.
+			</p>
+			
+			<h2> I don't know how to code, can I still get involved? </h2>
```

Comment:
*super* picky: it would be delightful if we could tighten this up in CSS:

<img width="587" alt="image" src="https://user-images.githubusercontent.com/1594827/199625872-73feed2e-fe51-4030-b1ac-52bd4d7c1a88.png">

Reply:
Yeah can do, one for tomorrow